components/ui/Navbar.js

```diff
@@ -19,41 +19,40 @@ const navbarItems = [
     link: '/nasa_collaboration',
   }
 ];
+{/* cretae navItem */}
+const NavItem = ({ title, link, isActive }) => {
+  return (
+    <li className={classNames(styles.navbarLinks, { [styles.isLinkActive]: isActive })}>
+      <Link href={link}>
+        <b>{navbarItems.findIndex((item) => item.link === link).toString().padStart(2, "0")}</b> {title}
```

Comment:
```suggestion
        <b>{(navbarItems.findIndex((item) => item.link === link) + 1).toString().padStart(2, "0")}</b> {title}
```

Otherwise you would index your pages from zero. Whilst that is quite the computer scientist creative touch, this page is about astrophysics. 😜